### PR TITLE
fix: misc µTP trace log fixes

### PR DESCRIPTION
**Pull request by tearfur**

### CODE REVIEW

**Review by reardonia — APPROVED** [review]

File: libtransmission/tr-udp.cc [hunk 1]
```diff
@@ -109,6 +109,8 @@ void event_callback(evutil_socket_t s, [[maybe_unused]] short type, void* vsessi
             return;
         }
 
+        auto const from_str = tr_socket_address::from_sockaddr(from_sa).value_or(tr_socket_address{}).display_name();
```
Comment (mikedld) [inline libtransmission/tr-udp.cc:112]: Only needed in error cases, and only if trace logs are enabled, maybe turn into a lambda / helper function (which may exist somewhere already?) for lazy evaluation?

Comment (tearfur) [reply libtransmission/tr-udp.cc]: dcb75f303d7eb5a8b2942d00bb0845b49e5cee68

**Comment by tearfur** [comment]
Rebased to `main`.

File: libtransmission/tr-udp.cc [hunk 2]
```diff
@@ -109,6 +109,11 @@ void event_callback(evutil_socket_t s, [[maybe_unused]] short type, void* vsessi
             return;
         }
 
+        auto const from_str = [from_sa]
```
Comment (mikedld) [inline libtransmission/tr-udp.cc:112]: Doesn't depend on anything inside the loop, could be moved out of `for (;;)`.

Comment (tearfur) [reply libtransmission/tr-udp.cc]: d39fde3c4c5771573621d0ba620dcc58e409f49f

**Review by mikedld — APPROVED** [review]

**Pull request merged**